Source/Core/Core/PowerPC/PPCAnalyst.h

```diff
@@ -189,11 +190,10 @@ class PPCAnalyzer
     // Requires JIT support to be enabled.
     OPTION_CONDITIONAL_CONTINUE = (1 << 0),
 
-    // If there is a unconditional branch that jumps to a leaf function then inline it.
+    // Try to inline unconditional branches.
```

Comment:
Make it clear to the reader that this also includes call/ret,

    // Try to inline unconditional branches/calls/returns.